tensorflow_lite_support/examples/task/text/desktop/nl_classifier_op_resolver.cc

```diff
@@ -0,0 +1,196 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow_lite_support/examples/task/text/desktop/nl_classifier_op_resolver.h"
```

Comment:
`examples/desktop/task` isn't the appropriate place to put this code. Could you move it over to `cc/task/text/utils/op_resolver_utils.h`?

Reply:
Alright, will remove it then.